pywbem_mock/_cimrepository.py

```diff
@@ -2168,32 +2197,34 @@ def ReferenceNames(self, **params):
             if iname.host is None:
                 iname.host = self.host
 
-        return self._return_assoc_tuple(rtn_names)
+        return rtn_names
 
     ####################################################################
     #
     #   References server request
     #
     ####################################################################
 
-    def References(self, **params):
+    def References(self, ObjectName, ResultClass=None, Role=None,
+                   IncludeQualifiers=None, IncludeClassOrigin=None,
+                   PropertyList=None):
         """
-        Implements a mock WBEM server responder for
+        Implements a CIM Repository responder for
         :meth:`~pywbem.WBEMConnection.References`
         """
 
         namespace = self.namespace
```

Comment:
See previous comment about `namespace = self.namespace`. Only this time, the namespace is (optionally) passed in ObjectName.